test/space_test.jl

```diff
@@ -234,6 +234,41 @@ end
         [[], [], [a], [], []]
     end
 
+    gridspace_periodic = ABM(Agent3, GridSpace((3, 3); periodic = true))
+    @test node_neighbors((1, 1), gridspace_periodic) == [(1, 3), (3, 1), (2, 1), (1, 2)]
```

Comment:
The tests fail whenever the periodic grid comes into play. This is because the order changes from [N, W, E, S] to [E, W, N, S]. Anyone got an idea why that happens?

This applies to tests at LOC 238, 241, 248 as well as 254, 260, 267.

Reply:
I'd have to take a deeper look, but my guess would be that we don't respect this order when grabbing the wrap-around cells. That's something we'll need to take care of if so. Should have some time for this tomorrow.